tests/test_meta_cat.py

```diff
@@ -89,6 +93,30 @@ def test_predict_spangroup(self):
 
         n_meta_cat.config.general.span_group = None
 
+    def test_z_bert_meta_cat(self):
```

Comment:
This is not a good way to test your changes.
You're changing the state of the class (changing the value of `MetaCATTests .meta_cat`). The order that the tests are run is not tied to the order in which the tests are written in the class. As such, this change to the state of the class could make the other tests in the class be run on the MetaCAT you've set in this test rather than the one that was set up in the `setUpClass` method.

I would recommend making a new class that extends `unittest.TestCase` that has its own state and deals with this new type only.